server/dist/src/main/resources/concord-server.conf

```diff
@@ -450,6 +450,9 @@ concord-server {
 
         # interval for the automatic group sync (interval)
         minAgeSync = "1 day"
+
+        # interval to delete disabled users (interval)
```

Comment:
If we want to disable `delete` old user.. then `disabledAge` should be set to some large?

Reply:
That's actually a good question. I also don't think that this feature should be enabled by default.

Reply:
> If we want to disable delete old user.. then disabledAge should be set to some large one?

just set `disabledAge` to null...

Reply:
> That's actually a good question. I also don't think that this feature should be enabled by default.
fixed. now this functionality disabled by default...